### PR TITLE
[onert/cpu] Restructure cpu backend directory

### PR DESCRIPTION
**Pull request by hseok-oh**
- Rename directory and namespace: kernel -> ops
- Remove operand directory and namespace

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: runtime/onert/backend/cpu/Tensor.h [hunk 1]
```diff
@@ -28,8 +28,6 @@ namespace backend
 {
 namespace cpu
 {
-namespace operand
-{
 
 class Tensor : public ITensor
```
Comment (glistening) [inline runtime/onert/backend/cpu/Tensor.h:32]: @hseok-oh Please update inclusion guard `__ONERT_BACKEND_CPU_OPERAND_TENSOR_H__` to `__ONERT_BACKEND_CPU_TENSOR_H__`.

Comment (hseok-oh) [reply runtime/onert/backend/cpu/Tensor.h:32]: @glistening I'll update all `ifdef` guard in cpu backend after this PR.

**Comment by hseok-oh** [comment]
@glistening @wateret I've rebased to resolve conflict. PTAL

**Comment by wateret** [comment]
> Remove operand directory and namespace

This is something that I wanted to do!

**Pull request merged**